frontend/public/index.html

```diff
@@ -3,7 +3,33 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <title>Technigo React App</title>
+    <title>Tika and Maria's Project Authentication</title>
+
+    <!-- Meta tags for facebook -->
+    <meta property="og:title" content="Tika and Maria's Project Authentication" />
+    <meta property="og:url" content="https://tika-and-marias-project-autho.netlify.app/" />
+    <meta property="og:image" content="https://tika-and-marias-project-autho.netlify.app/images/ogimage.png" />
+    <meta property="og:image:width" content="1200" />
+    <meta property="og:image:height" content="630" />
+    <meta property="og:description" content="Register and read a message from a witch. Let's the day begin!" />
+    <meta property="og:locale" content="en" />
+
+    <!-- Meta tags for twitter -->
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta property="twitter:domain" content="tika-and-marias-project-autho.netlify.app/" />
+    <meta property="twitter:url" content="https://tika-and-marias-project-autho.netlify.app/" />
+    <meta name="twitter:title" content="Tika and Maria's Project Authentication" />
+    <meta name="twitter:description" content="Register and read a message from a witch. Let's the day begin!" />
+    <meta name="twitter:image" content="https://tika-and-marias-project-autho.netlify.app/images/ogimage.png" />
+
+    <!-- Favicon -->
```

Comment:
Cool with favicon added. We cant understand the difference, would you like to explain it to us?